brainfuck/brainfuck_test.go

```diff
@@ -0,0 +1,152 @@
+package brainfuck
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+// Test of the interpreter
+func TestInterpret(t *testing.T) {
+	rescueStdout := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	Interpret("+++++c>p++++++[-<+>]<.")
+
+	w.Close()
+	out, _ := ioutil.ReadAll(r)
+	os.Stdout = rescueStdout
+
+	if out[0] != 16 {
```

Comment:
why you compare only count of symbols? 
Pls check the content as well

Reply:
I tried to debug this piece of code, and it checks the content correctly.
Maybe, I haven't understood your advice, but `out` is a slice of `byte[]` and, during the debug, the only representation of this variable I received, was `[16]`

This is the proof:
![12](https://user-images.githubusercontent.com/40440883/161162993-eb79bb8d-5b89-4e14-9b28-8cc1ec6b56c2.jpg)

Reply:
yes. that's ok.